uint64 gid

time stamp
geometry_msgs/Pose estimate
sensor_msgs/PointCloud2 cloud